lib/vm/opFns.js

```diff
@@ -1082,15 +1082,15 @@ function isCreateOpCode (opName) {
 }
 
 function getContractStorage (runState, address, key, cb) {
-  if (runState._common.activeHardfork() === 'constantinople') {
+  if (runState._common._hardfork === 'constantinople') {
```

Comment:
Why is this switching from using supported API functionality to internal variable access? This should very much be avoided.

Reply:
(I was actually referring to `activeHardfork()` method used in some in-between state).

To get the hardfork there is an [API method](https://github.com/ethereumjs/ethereumjs-common/blob/master/src/index.ts#L375) in Common.

Reply:
@holgerd77 yeah, that was a bad mistake. Fixed now.